crd.yaml

```diff
@@ -68,6 +68,9 @@ spec:
                     type: string
                   property:
                     description: Property to extract if secret in backend is a JSON object
+                  isBinary:
+                    description: Handle the secret as a binary file, used in Azure KeyVault to avoid base64 encoding a binary file as they are already base64 encoded.
```

Comment:
nit: I'd rephrase this a bit to be more prescriptive for casual users.

```
You must set this to true if configuring an item for a binary file stored in Azure KeyVault.  Azure automatically base64 encodes binary files and setting this to true ensures External Secrets does not base64 encode the base64 encoded binary files.
```

Reply:
Sure thing, I modified the description to what you wrote here